### PR TITLE
i18n(fr): update `integrations-guide/db.mdx`

### PR DESCRIPTION
**Pull request by ArmandPhilippot**
#### Description (required)

Removes Astro Studio references in the French translation of `integrations-guide/db.mdx` (see #9509)

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Comment by yanthomasdev** [comment]
Almost good to go, there's a small link issue, could you take a look? 🙌

File: src/content/docs/fr/guides/integrations-guide/db.mdx [hunk 1]
```diff
@@ -214,18 +214,18 @@ Vérifier s'il y a des différences entre la configuration de la base de donnée
...
 
-Exécute un fichier `.ts` ou `.js` pour lire ou écrire dans votre base de données. Cette commande accepte un chemin de fichier comme argument, et supporte l'utilisation du module `astro:db` pour écrire des requêtes sûres. Utilisez l'option `--remote` pour lancer l'exécution sur la base de données de votre projet Studio, ou ignorez l'option pour lancer l'exécution sur votre serveur de développement. Voir comment utiliser les [données sur le développement des seeds](/fr/guides/astro-db/#alimentez-votre-base-de-données-pour-le-développement) pour un exemple de fichier.
+Exécute un fichier `.ts` ou `.js` pour lire ou écrire dans votre base de données. Cette commande accepte un chemin de fichier comme argument, et supporte l'utilisation du module `astro:db` pour écrire des requêtes sûres. Utilisez l'option `--remote` pour exécuter sur votre base de données compatible libSQL, ou ignorez l'option pour lancer l'exécution sur votre serveur de développement. Voir comment utiliser les [données sur le développement des seeds](/fr/guides/astro-db/#alimentez-votre-base-de-données) pour un exemple de fichier.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/integrations-guide/db.mdx:219]:
```suggestion
Exécute un fichier `.ts` ou `.js` pour lire ou écrire dans votre base de données. Cette commande accepte un chemin de fichier comme argument, et supporte l'utilisation du module `astro:db` pour écrire des requêtes sûres. Utilisez l'option `--remote` pour exécuter sur votre base de données compatible libSQL, ou ignorez l'option pour lancer l'exécution sur votre serveur de développement. Voir comment utiliser les [données sur le développement des seeds](/fr/guides/astro-db/#alimentez-votre-base-de-données-pour-le-développement) pour un exemple de fichier.
```

**Comment by ArmandPhilippot** [comment]
@yanthomasdev Done! I suspected there was going to be a conflict when I edited this file again today... 😅

**Pull request merged**